docs/api.md

```diff
@@ -40,9 +40,8 @@ separator.update_parameter(segment=smaller_segment)
 ```python
 # Remember to create the destination folder before calling `save_audio`
 # Or you are likely to recieve `FileNotFoundError`
```

Comment:
Please also replace "recieve" with receive, thanks

Reply:
Hi, fixed in new commit.